packages/main/src/Breadcrumbs.ts

```diff
@@ -176,19 +166,16 @@ class Breadcrumbs extends UI5Element {
 	 *
 	 * <br><br>
 	 * <b>Note:</b> Use the <code>ui5-breadcrumbs-item</code> component to define the desired items.
-	 * @type {sap.ui.webc.main.IBreadcrumbsItem[]}
-	 * @name sap.ui.webc.main.Breadcrumbs.prototype.default
-	 * @slot items
 	 * @public
 	 */
 	@slot({ type: HTMLElement, invalidateOnChildChange: true, "default": true })
-	items!: Array<BreadcrumbsItem>;
+	items!: Array<IBreadcrumbsItem>;
 
```

Comment:
BreadcrumbsItem is an abstract component so we can use class instead of interface (interface is redundant)